recipes-ni/sysconfig-settings/sysconfig-settings_1.0.bb

```diff
@@ -141,10 +143,16 @@ pkg_postinst_ontarget_${PN} () {
 		ln -sf ${settingsdatadir}/fpga_target.ini ${systemsettingsdir}/fpga_target.ini
 	fi
 
+	# add console out if we have a firmware variable for it (x86_64 targets only)
+	efiConsoleOutEnable=$(/sbin/fw_printenv -n BootFirmwareConsoleOutEnable 2>/dev/null || true)
```

Comment:
Can we reliably use `PATH` to find `fw_printenv` here (and on line 133 in this file), and add an RDEPENDS on the `fw-printenv` package?